### PR TITLE
Created partners page

### PR DESCRIPTION
**Pull request by hathix**
By @aidan-fitz

### CODE REVIEW

**Comment by hathix** [comment]
@aidan-fitz , this looks great! Just one bug I found:

screen shot 2017-03-21 at 4 31 37 pm

I think the error comes in this line of HTML:

```
<img class="media-object img-circle mini-media" src="/assets/images/partners/{{ partner.image }}" alt="{{ partner.name}}, {{ partner.college }}">
```

`partner.college` is an object, so if you try putting it into `alt`, it goes crazy. So I think you can just change `partner.college` to `partner.college.full`, which is a string and will work well in the `alt` attribute.

**Comment by hathix** [comment]
Also, do we need to have the list of partners on the index page *and* a dedicated partners page? If you do want to duplicate it, it'd make sense to put an `<a>` from the index page to the partners page.

**Comment by aidan-fitz** [comment]
Hm, maybe the first three organizations on the index page. I'm also adding the other partners now.

File: _data/partners.yml [hunk 1]
```diff
@@ -1,8 +1,23 @@
 -   name: Harvard Open Data Project
     url: http://harvard-open-data-project.github.io/
-    college: Harvard
+    college:
+      short: Harvard
+      full: Harvard University
     image: harvard-open-data-project.png
```
Comment (hathix) [inline _data/partners.yml]: You can include my information here as the contact! My email is neelmehta@college.harvard.edu.

File: _data/partners.yml [hunk 2]
```diff
@@ -10,3 +10,11 @@
...
       full: Princeton University
     image: envision.png
+-   name: Code4Good
+    college:
+      short: Yale
+      long: Yale University
+-   name: Coding it Forward Cornell
+    college:
+      short: Cornell
+      long: Cornell University
```
Comment (hathix) [inline _data/partners.yml]: There's an inconsistency in the naming here! Here, `college` has `long` and `short`, but before it has `full` and `short`. That's causing the webpage to render inconsistently.

File: _data/partners.yml (hunk 1 above)
Comment (hathix) [inline _data/partners.yml]: This image isn't found. Use `hodp-logo-square.png` instead.

File: _data/partners.yml (hunk 2 above)
Comment (aidan-fitz) [inline _data/partners.yml]: Oh yeah, just caught that. Thanks.

File: _data/partners.yml (hunk 1 above)
Comment (aidan-fitz) [inline _data/partners.yml]: Ah, I meant to type `harvard-open-data-project-logo.png`. But yeah, the square one is better.

**Comment by aidan-fitz** [comment]
I just made the corrections. Thanks!

**Comment by hathix** [comment]
Looks good!

**Pull request merged**